spacemesh/v1/mesh_types.proto

```diff
@@ -107,3 +107,21 @@ message EpochStreamRequest {
 message EpochStreamResponse {
     ActivationId id = 1;
 }
+
+message MalfeasanceRequest {
+    SmesherId smesher = 1;
+    string smesher_hex = 2;
```

Comment:
Are these both encoding the NodeID? Can we remove one of the two (otherwise what would happen if they are not the same and both valid?)

Reply:
thanks. i think i will keep the hex version for UX.

Reply:
I would have gone with the other one for consistency 😅, but I'm OK with either as long as we only have one.

Reply:
so the machines (explorer/dashboard/smapp) will use the streaming API.
the query is mainly for human, hence the choice.